standalone/src/models/shipping-method/zone-rate/fields-config.ts

```diff
@@ -0,0 +1,39 @@
+import { fake, type TModelFieldsConfig } from '@/core';
+import {
+  KeyReference,
+  ReferenceGraphql,
+  TReferenceGraphql,
+} from '@/models/commons';
+import { Zone } from '@/models/zone';
+import { ShippingRateRest } from '../shipping-rate';
+import type { TZoneRateRest, TZoneRateGraphql } from './types';
+
+const commonFieldsConfig = {
+  shippingRates: fake(() => [ShippingRateRest.random()]),
```

Comment:
This can't be populated in the common fields as each fields config should use its corresponding nested model: `ShippingRateRest | ShippingRateGraphql`